configs/setup_golden_image.py

```diff
@@ -183,23 +169,24 @@ def main(args):
     assert os.geteuid() == 0, 'This script requires root permissions'
 
     # -- Build 'raw' golden image using vmdebootstrap --
+    # Skip bootstrap when --image param is used
     if not getattr(args, 'skip_bootstrap', False):
-        #vmd_path = args.extra[1] if len(args.extra) >= 2 else None
-        vmd_path = getattr(args, 'vmd_cfg', None)
-        debootstrap_image(args.config, vmd_path=vmd_path)
+        if getattr(args, 'image', None) is None:
+            return
+            vmd_path = getattr(args, 'vmd_cfg', None)
+            debootstrap_image(args.config, vmd_path=vmd_path)
     else:
-        print (' -- Skipping bootstrap stage, since --skip-bootstrap param used.')
+        print (' -- Skipping bootstrap stage. --skip-bootstrap or --image is used.')
 
-    if getattr(args, 'golden', False) is not False:
-        download_image(args.golden)
+    if getattr(args, 'image', False) is not False:
```

Comment:
Not pythonic.  Use "if getattr(args, 'image', False):"

and "image" is far too generic a name.